src/app/MD/page.jsx

```diff
@@ -111,6 +112,14 @@ export default function MarkdownEditor() {
     localStorage.setItem("theme", JSON.stringify(newTheme));
   };
 
+  const copyToClipboard = async () => {
+    try {
+      await navigator.clipboard.writeText(markdown);
```

Comment:
When you click it i want to have a notification of that it was copied using a react toaster. You can the toaster code from this https://github.com/Bashamega/WebDevTools/blob/main/src/app/customizer/image-color-picker/page.jsx